gax-java/gax/src/test/resources/META-INF/native-image/com.google.api/gax/native-image.properties

```diff
@@ -0,0 +1,8 @@
+Args=--initialize-at-build-time=java.lang.annotation.Annotation,\
+  org.junit.experimental.categories.Category,\
+  org.junit.experimental.categories.CategoryValidator,\
+  org.junit.Ignore,\
+  org.junit.runner.RunWith,\
+  org.junit.runners.model.FrameworkField,\
+  org.junit.validator.AnnotationValidator,\
+  org.junit.vintage.engine.discovery.FilterableIgnoringRunnerDecorator
```

Comment:
Is this only adding the junit configs?

It's been a while since my release, but I recall storage-nio required some custom configurations: https://github.com/googleapis/java-storage-nio/blob/97c3624ac7e375621d78c7eea10b00359d7971fd/google-cloud-nio/src/main/resources/META-INF/native-image/com/google/cloud/google-cloud-nio/native-image.properties#L7-L12 

I think one missing from there is guava's `com.google.common.collect.RegularImmutableList`

Reply:
I added the relevant config and raise a PR to be merged once this one is released: https://github.com/googleapis/java-storage-nio/pull/1559